documentation/Installation/PHP.md

```diff
@@ -235,7 +235,7 @@ brew install -v php71-opcache
 
 ```bash
 brew unlink php71 && brew link php72
-brew install -v php71-opcache
+brew install -v php72-opcache
```

Comment:
Opcode caching is (in the homebrew-core version of PHP) automatically installed. No need to install it anymore.